packages/bruno-app/src/utils/collections/index.js

```
@@ -406,7 +406,7 @@ export const transformCollectionToSaveToExportAsFile = (collection, options = {}
             };
             break;
           case 'oauth2':
-            let grantType = get(si.request, 'auth.oauth2.grantType', '');
+            const grantType = get(si.request, 'auth.oauth2.grantType', '');
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)